content_scripts/vimium_frontend.coffee

```diff
@@ -94,7 +94,12 @@ settings =
 #
 # Give this frame a unique id.
 #
-frameId = Math.floor(Math.random()*999999999)
+window.frameId = Math.floor(Math.random()*999999999)
```

Comment:
The assignment to window is already done at the bottom of the file; this change is unnecessary.